namer/consul/src/main/scala/io/buoyant/namer/consul/SvcAddr.scala

```diff
@@ -113,11 +126,11 @@ private[consul] object SvcAddr {
             )
             Future.exception(NoIndexException)
 
-          case Return(v1.Indexed(addrs, xConsulIndex)) =>
+          case Return(v1.Indexed((srvMeta, addresses), xConsulIndex)) =>
             stats.updates.incr()
-            val addr = addrs match {
+            val addr = addresses match {
               case addrs if addrs.isEmpty => Addr.Neg
-              case addrs => Addr.Bound(addrs, meta)
+              case addrs => Addr.Bound(addrs, meta ++ Map(Metadata.serviceMeta -> srvMeta))
```

Comment:
I don't think we need to bury the service meta under a "serviceMeta" key.  We can just do `meta ++ srvMeta`.